internal/engine/synclet_build_and_deployer.go

```diff
@@ -50,6 +50,10 @@ func (sbd *SyncletBuildAndDeployer) BuildAndDeploy(ctx context.Context, manifest
 	span.SetTag("manifest", manifest.Name.String())
 	defer span.Finish()
 
+	if manifest.IsDockerCompose() {
+		return store.BuildResult{}, RedirectToNextBuilderf("not implemented: DC container builds")
```

Comment:
```suggestion
		return store.BuildResult{}, RedirectToNextBuilderf("not implemented: DC synclet builds")
```

Reply:
technically a synclet built is a subset of container builds, and NO container builds are supported, but sure :P